app/api/private-cloud/products/[licencePlate]/comments/_operations/create.ts

```diff
@@ -1,12 +1,27 @@
 import prisma from '@/core/prisma';
 
-export async function createOp(text: string, projectId: string, userId: string) {
-  const comment = await prisma.privateCloudComment.create({
-    data: {
-      text,
-      userId,
-      projectId,
+export async function createOp(text: string, userId: string, projectId?: string, requestId?: string) {
```

Comment:
let's make `productId` required and `requestId` optional.

Reply:
this would then throw an error on a `create request` as there is no `projectId` to provide...